docs/guides/payments.rst

```diff
@@ -36,6 +36,14 @@ Example
         client_token = gateway.client_token.generate()
         return client_token
 
+
+.. note::
+
+    All the below methods receive payment_information as a dataclass - PaymentData. 
+    Methods should return a response as a dataclass - GatewayResponse. 
+    Description of the given structures can be found below.
```

Comment:
```suggestion
    The description of the given structures can be found below.
```

Reply:
About this change, let's use Grammarly to make sure about the grammar, it's available in the free plan.

Reply:
I always check it with Grammarly